.gitignore

```diff
@@ -12,6 +12,7 @@
 *.res
 .DS_Store
 .vs
+.vscode
```

Comment:
Remove this change:
```suggestion
```
(probably as other devs are using VSCode, we'll add this, but we can remove it from this PR)